app/code/Magento/Quote/Model/QuoteManagement.php

```diff
@@ -377,7 +377,7 @@ public function placeOrder($cartId, PaymentInterface $paymentMethod = null)
         $this->checkoutSession->setLastOrderStatus($order->getStatus());
 
         $this->eventManager->dispatch('checkout_submit_all_after', ['order' => $order, 'quote' => $quote]);
-        return $order->getId();
+        return $order->getIncrementId();
```

Comment:
This change is incorrect: https://github.com/magento/magento2/blob/65a8c75ba70bcb446865ce183e9d5b3c020d51ed/app/code/Magento/Quote/Api/CartManagementInterface.php#L64

Please apply a fix in a proper place.